src/OceanSeaIceModels/CrossRealmFluxes/similarity_theory_turbulent_fluxes.jl

```diff
@@ -381,24 +407,42 @@ end
     q₀ = AtmosphericThermodynamics.vapor_specific_humidity(ℂ, 𝒬₀)
```

Comment:
This isn't taking into account `θ₀` is it? This is the main effect of the prognostic temperature --- it changes the saturation vapor pressure thereby impacting latent heat fluxes

Reply:
You have to rebuild the thermodynamic state with the new temperature

Reply:
or actually just recompute it, I think there is a function for this

Reply:
oh right! good find